optashift-employee-rostering-webapp/src/test/java/org/optaplanner/openshift/employeerostering/webapp/skill/SkillRestServiceIT.java

```diff
@@ -0,0 +1,82 @@
+package org.optaplanner.openshift.employeerostering.webapp.skill;
+
+import java.util.Random;
+import javax.ws.rs.core.Response;
+import org.junit.Before;
+import org.junit.Test;
+import org.optaplanner.openshift.employeerostering.shared.skill.Skill;
+import org.optaplanner.openshift.employeerostering.shared.skill.SkillRestService;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.List;
+
+public class SkillRestServiceIT extends AbstractRestServiceIT {
+
+    private static final int TENANT_ID = new Random().nextInt(Integer.MAX_VALUE);
```

Comment:
The random is unseeded.
This is a randomly failing test that fails if the nextInt is the same as a tenant id in the auto generated testdata.
Just hard corde TENANT_ID -1 or -1000 instead.

Reply:
The test will not necessarily fail in case it jumps into tenant ID that already has some data associated with - the @Before method clears the Skill data in such a case. On the other hand I agree the random here is needless complication, will remove it.